velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -184,56 +180,29 @@ VectorFuzzer::Options getFuzzerOptions() {
 
 std::optional<CallableSignature> processSignature(
     const std::string& functionName,
+    const std::vector<TypePtr>& argTypes,
     const exec::FunctionSignature& signature) {
-  // Don't support functions with parameterized signatures.
-  if (!signature.variables().empty()) {
-    LOG(WARNING) << "Skipping unsupported signature: " << functionName
-                 << signature.toString();
-    return std::nullopt;
-  }
-  if (signature.variableArity() && !FLAGS_enable_variadic_signatures) {
-    LOG(WARNING) << "Skipping variadic function signature: " << functionName
-                 << signature.toString();
-    return std::nullopt;
-  }
-
   CallableSignature callable{
       .name = functionName,
-      .args = {},
+      .args = argTypes,
       .variableArity = signature.variableArity(),
       .returnType =
           SignatureBinder::tryResolveType(signature.returnType(), {}, {})};
   VELOX_CHECK_NOT_NULL(callable.returnType);
 
   bool onlyPrimitiveTypes = callable.returnType->isPrimitiveType();
 
-  // Process each argument and figure out its type.
-  for (const auto& arg : signature.argumentTypes()) {
-    auto resolvedType = SignatureBinder::tryResolveType(arg, {}, {});
-
-    // TODO: Check if any input is Generic and substitute all
-    // possible primitive types, creating a list of signatures to fuzz.
-    if (!resolvedType) {
-      LOG(WARNING) << "Skipping unsupported signature with generic: "
-                   << functionName << signature.toString();
-      return std::nullopt;
-    }
-
-    onlyPrimitiveTypes &= resolvedType->isPrimitiveType();
-    callable.args.emplace_back(resolvedType);
+  for (const auto& arg : argTypes) {
+    onlyPrimitiveTypes &= arg->isPrimitiveType();
```

Comment:
nit: &= is equivalent to using bitwise AND not the logical AND (&&) operator. While this is harmless in this case, I would still recommend using the logical AND operator for better readability (that communicates that a logical operation is intended here). Since there is no shortcut for it, onlyPrimitiveTypes =  onlyPrimitiveTypes && arg->isPrimitiveType(); will suffice